apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessDialog.tsx

```diff
@@ -0,0 +1,416 @@
+      }
+      securityClearance {
+        value
+        label {
+          localized
+        }
+      }
+      processNumber
+    }
+  }
+`);
+
+interface ReviewRecruitmentProcessDialogProps {
+  recruitmentProcessQuery: FragmentType<
+    typeof ReviewRecruitmentProcessDialog_Fragment
+  >;
+}
+
+const ReviewRecruitmentProcessDialog = ({
+  recruitmentProcessQuery,
+}: ReviewRecruitmentProcessDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+
+  const recruitmentProcess = getFragment(
+    ReviewRecruitmentProcessDialog_Fragment,
+    recruitmentProcessQuery,
+  );
+
+  const pool = recruitmentProcess?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName =
+    pool.name?.localized ?? intl.formatMessage(commonMessages.notFound);
+
+  const [, executeMutation] = useMutation(ReviewRecruitmentProcess_Mutation);
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const isSuspended = !!recruitmentProcess.suspendedAt;
+  const methods = useForm<FormValues>({
+    defaultValues: { isSuspended: isSuspended ? "true" : "false" },
+  });
+  const {
+    handleSubmit,
+    formState: { isSubmitting },
+  } = methods;
+
+  const updateSuspendedAtStatus = async (values: FormValues) => {
+    await executeMutation({
+      id: recruitmentProcess.id,
+      isSuspended: values.isSuspended === "true",
+    })
+      .then((res) => {
+        if (!res.error) {
+          setIsOpen(false);
+          if (values.isSuspended === "true") {
+            toast.success(
+              intl.formatMessage({
+                defaultMessage:
+                  "You have been removed from the search results.",
+                id: "PoFTwr",
+                description:
+                  "Alert displayed to the user when application card dialog submits successfully.",
+              }),
+            );
+          } else {
+            toast.success(
+              intl.formatMessage({
+                defaultMessage:
+                  "You have been added back into the search results.",
+                id: "lB/SWR",
+                description:
+                  "Alert displayed to the user when they updated something to appear in search results again.",
+              }),
+            );
+          }
+        }
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: failed removing you from search results.",
+            id: "7tdU/G",
+            description:
+              "Alert displayed to the user when application card dialog fails.",
+          }),
+        );
+      });
+  };
+
+  const status = getQualifiedRecruitmentStatusChip(
+    recruitmentProcess.suspendedAt,
+    recruitmentProcess.placedAt,
+    intl,
+  );
+
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
+      <Dialog.Trigger asChild>
+        <PreviewList.Button
+          label={poolName}
```

Comment:
Same thing about adding the additional context for the button here.

Reply:
Added in here [e9fa102](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/e9fa10262e89daca332eefdd9ec54cc02242e5e8)